Source/WebCore/rendering/RenderLayerFilters.h

```diff
@@ -78,7 +78,7 @@ class RenderLayerFilters final : private CachedSVGDocumentClient {
     void notifyFinished(CachedResource&, const NetworkLoadMetrics&, LoadWillContinueInAnotherProcess) final;
     void resetDirtySourceRect() { m_dirtySourceRect = LayoutRect(); }
 
-    RenderLayer& m_layer;
+    CheckedRef<RenderLayer> m_layer;
```

Comment:
You should make this const too. Then we won't need to add stack variables for this in RenderLayerFilters.cpp.